ci/ci/envoy.py

```diff
@@ -172,13 +172,17 @@ def auth_check_exemption() -> dict:
     }
 
 
-def rate_limit_config() -> dict:
+def rate_limit_config(service: str, namespace: str) -> dict:
+    # FIXME We didn't set a rate limit before with nginx so I'm making this
+    # sort of unreasonably high. This should be tuned per service
```

Comment:
Good point, I forgot about our pod autoscalers and have just been thinking 1-3 pods. The limit is per service but also per internal-gateway/gateway pod, so at least 3x the given number (and possibly more from the gateway since we allow that to autoscale).